changelogs/fragments/wait_condition.status_is_a_str.yaml

```diff
@@ -0,0 +1,3 @@
+---
+minor_changes:
```

Comment:
Can you rename the fragment to `310-wait_condition.status_is_a_str.yaml` (just to keep similar format to all the others with issue/PR in the name).

Reply:
Done.